docs/source/yelpsoa_configs.rst

```diff
@@ -51,6 +51,13 @@ instance MAY have:
     namespace. For example ``canary`` instances can have ``nerve_ns: main`` to route
     their traffic to the same pool as the other ``main`` instances.
 
+  .. _networking_mode:
+
+  * ``networking_mode``: Specify which kind of 
```

Comment:
@solarkennedy done. `'host'` and `'bridge'` become `'HOST'` and `'BRIDGE'` just before they're passed to marathon/chronos. If a service author gives a net string that's not none, host or bridge it just passes that string straight to marathon/chronos.